neural_tangents/utils/kernel.py

```diff
@@ -17,11 +17,12 @@
 
 import operator as op
 from typing import Dict, Tuple, Optional, Callable, Any
-
-import jax.numpy as np
 from neural_tangents.utils import dataclasses
 from neural_tangents.utils import utils
 
+import tensorflow as tf
+from tensorflow.python.ops import numpy_ops as np
+from tf_helpers.bitwise import bitwise_or
 
 @dataclasses.dataclass
```

Comment:
Is there really no problem here? dataclasses.dataclass registers Kernel as a JAX PyTree node, which is required to get desired behavior around static functions / jit.

Reply:
Great point, Sam! Thanks for bringing this up!

Reply:
By the way, Sam @sschoenholz , if I remove the `dataclasses` wrapper, it seems that the `Kernel` object initialization does not require any parameters at all?

Reply:
Yes, that's unfortunately a problem, since the `dataclass` decorator gave it the necessary initialization method (i.e. just filling all the fields with passed values). So without it you'd need to create your own `__init__` for the code to work I think. But see my other comment, I think we can keep it as a dataclass

Reply:
I think another option would be to use Python's dataclasses.dataclass rather than the NT version.

Reply:
Yes, Sam, I did add Python’s native dataclasses!

Reply:
Yes, Sam, I did add Python’s native dataclasses!